src/components/staking/PageValidator.vue

```diff
@@ -9,13 +9,27 @@
     class="small"
   >
     <template v-if="validator.operatorAddress" slot="managed-body">
-      <div class="status-container">
-        <span :class="validator.status | toLower" class="validator-status">
-          {{ validator.status }}
-        </span>
-        <span v-if="validator.statusDetailed" class="validator-status-detailed">
-          {{ validator.statusDetailed }}
-        </span>
+      <button
+        class="validators-list-button"
+        color="secondary"
+        @click="$router.push(`/validators`)"
+      >
+        <img src="/img/icons/arrow-icon.svg" />
```

Comment:
@Bitcoinera @faboweb we already import material icons for this. we don't need to include separate svg's for this kind of stuff.

Reply:
Charlie seemed very happy with this arrow. I don't know :woman_shrugging:

Reply:
sure. but we already import them as fonts - icons should be used like this (not imported as separate SVGs):

https://github.com/luniehq/lunie/blob/b9566b9f842c7060378bae337c134aa32896eaaf/src/components/transactions/message-view/SendMessageDetails.vue#L24

Reply:
pushed: https://github.com/luniehq/lunie/pull/3388